### PR TITLE
add ability for staff to freeze and unfreeze

### PR DESCRIPTION
**Pull request by BrandonSharratt**
*Issue:*https://github.com/bcgov/entity/issues/21786

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-47-6ifdrref.web.app

**Review by hfekete — APPROVED** [review]
Nice work!
Had only one comment, that can either be actioned or not, depending on your answer.

File: src/components/bcros/filing/AddStaffFiling.vue [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+          filingError.value = false
+          business.loadBusiness(currentBusiness.value.identifier, true)
+          filings.loadFilings(currentBusiness.value.identifier, true)
+          isModalOpen.value = false
+        }
+      }
+    }]
+})
+
+const actions: ComputedRef<Array<Array<MenuActionItem>>> = computed(() => {
```
[the remark this answers is not in the document]
Comment (BrandonSharratt) [reply src/components/bcros/filing/AddStaffFiling.vue:66]: Yes, it won't right now but eventually that menu will be expanded to match the options currently available (many are in the more actions tombstone, like consent to amalgam out)

[the remark this answers is not in the document]
Comment (hfekete) [reply src/components/bcros/filing/AddStaffFiling.vue:66]: Good to merge!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-47-6ifdrref.web.app

**Pull request merged**